tests/ctl/core/test_api.py

```diff
@@ -156,6 +158,93 @@ def test_api_delete(
         print(result.text)
         assert result.status_code == 200
 
+    @pytest.mark.parametrize("endpoint", TAXONOMY_ENDPOINTS)
+    def test_api_cannot_modify_default_taxonomy(
+        self, test_config: FidesctlConfig, endpoint: str
+    ) -> None:
+        resource = getattr(DEFAULT_TAXONOMY, endpoint)[0]
+
+        result = _api.update(
+            url=test_config.cli.server_url,
+            headers=test_config.user.request_headers,
+            resource_type=endpoint,
+            json_resource=resource.json(exclude_none=True),
+        )
+        assert result.status_code == 403
```

Comment:
Multiple asserts in a single test can get really troublesome to debug (it fails on the first assert)

Can you split these into discrete tests?

Reply:
these might even make sense as their own test class, but I have less strong opinions there